global: fix build failure

https://travis-ci.org/inveniosoftware/invenio-csl-rest/jobs/656679109
```
    def __getattr__(self, name):
        if name == "__members__":
            return dir(self._get_current_object()
>       return getattr(self._get_current_object(), name)
E       AttributeError: 'Request' object has no attribute 'is_xhr'
```
Fix also Travis deprecation:
https://travis-ci.org/inveniosoftware/invenio-csl-rest/jobs/656679109/config

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,8 +26,6 @@
 notifications:
   email: false
 
-sudo: false
-
 language: python
 
 cache:
@@ -39,9 +37,8 @@ env:
   - REQUIREMENTS=devel
 
 python:
-  - "2.7"
-  - "3.5"
-  - "pypy"
+  - "3.6"
+  - "3.7"
 
 matrix:
   allow_failures:
@@ -72,5 +69,5 @@ deploy:
   distributions: "sdist bdist_wheel"
   on:
     tags: true
-    python: "2.7"
+    python: "3.6"
     condition: $REQUIREMENTS = release

--- a/requirements-devel.txt
+++ b/requirements-devel.txt
@@ -22,3 +22,4 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 -e git+https://github.com/inveniosoftware/invenio-rest.git#egg=invenio-rest
+-e git+https://github.com/inveniosoftware/invenio-base.git#egg=invenio-base

--- a/setup.py
+++ b/setup.py
@@ -59,7 +59,7 @@ setup_requires = [
 ]
 
 install_requires = [
-    'Flask>=0.11.1',
+    'invenio-base>=1.2.2',
     'citeproc-py-styles>=0.1.0',
     'invenio-rest>=1.0.0a9',
 ]

